### PR TITLE
Add report api

### PR DESCRIPTION
**Pull request by anetteu**

### CODE REVIEW

File: backend/src/flotilla/api/reports_api.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    tags=["Reports"],
+    summary="List all available reports on the asset",
+    dependencies=[Security(authentication_scheme)],
+)
+async def get_reports(
+    db: Session = Depends(get_db),
+) -> List[Report]:
+    """List all available reports on the asset."""
+    db_reports: List[ReportDBModel] = read_reports(db)
+    logger.info(f"{db_reports}")
```
Comment (vetlek) [inline backend/src/flotilla/api/reports_api.py:39]: Probably not necessary to log both the reports in the database and the returned reports. Suggest that both are removed until we see the need

**Review by vetlek — CHANGES_REQUESTED** [review]
Looks good. One small change

File: backend/src/flotilla/api/reports_api.py (hunk 1 above)
Comment (anetteu) [reply backend/src/flotilla/api/reports_api.py:39]: You are right, this is dev logging that I forgot to remove

**Review by vetlek — APPROVED** [review]
LGTM

**Pull request merged**